### DISCUSSION TITLE
Bad input to API call

### DISCUSSION THREAD

**Discussion by theandrewwade** [samschott/maestral#756] [Q&A] [answered]
Hi all,

I've today started getting this error. After the error it pauses syncing, so I hit "Resume Sync".

I have tried to Rebuild Index, and have quit and restarted Maestral, and rebooted the machine. No change.

It so far has only done this for folders that are marked to not sync.

Any ideas?

![CleanShot 2022-11-24 at 12 00 30](https://user-images.githubusercontent.com/106374662/203755128-eac6e8f0-b2a6-4eac-9d21-e51b80331061.png)

**Comment by kimralphio** [comment]
Hi @theandrewwade,

I also started receiving this error as of today.

I am receiving it for folders/files that **are** marked to sync.

No solution for you! Just wanted to raise it as well.

Thanks!

**Reply by theandrewwade** [reply]
Thanks! Perhaps I just haven't got to those folders yet 😬 Pretty large repo.

**Reply by theandrewwade** [reply]
Ah! Now I'm getting errors on synced folders as well.

**Comment by robot1125** [comment]
I'm new to Maestral, and I'm getting the same error on a fresh install.

**Comment by joekissell** [comment]
I'm getting these too, for many folders regardless of sync status. From a quick web search it looks as though other third-party Dropbox clients are getting the same error. It appears to be some sort of API breakage on Dropbox's side, though it's unclear whether it was an intentional change or a bug.

**Comment by samschott** [comment, marked as answer]
This is an issue on the Dropbox side which should be resolved now, though the rollout may take some time to reach all servers.

**Reply by theandrewwade** [reply]
Thanks Sam. I haven't had the error again since last night.
